rpm/SOURCES/etc/pepProxy/README.md

```diff
@@ -0,0 +1,29 @@
+# PEP Proxy configuration procedure
+
+PEP Proxy is able to start multiple instances by adding and configuring certain files in `/etc/pepProxy`
+and using `pepProxy` service script
+
+To start multiple instances is needed to place in `/etc/pepProxy` one configuration file for 
+each instance that is wanted to run. RPM cames with one preconfigured instance (config 
+file called pepproxy_default.conf) that can be used as a template to configure another instances.
+
+The `pepProxy` service script to start looks for files in `/etc/pepProxy` that begins with `pepproxy_` 
+prefix and has `.conf` extension and start (or stop or status or restat) one process for file found.
+
+It is important to change `PROXY_PORT` and `ADMIN_PORT` to one not used by other PEP intances/services. 
+
+`pepProxy` init.d is packaged into the RPM and is needed to execute PEP Proxy 
+in multiinstace explained above. It has the next operations:
+- **start**: `sudo /sbin/service pepProxy start [<instance>]` if `<instance>` is not provided, script try to
```

Comment:
Rewrite it a little bit, please. Suggestion:

"If <instance> is not provided, the script tries to start as many instances as found in the configuration folder as possible (matching the configuration file pattern). Otherwise, it only starts a single instance with the provided name"